xml/System.Runtime.CompilerServices/Unsafe.xml

```diff
@@ -531,10 +643,10 @@
         </Parameter>
       </Parameters>
       <Docs>
-        <typeparam name="T">The type of reference.</typeparam>
+        <typeparam name="T">The underlying type of the reference.</typeparam>
         <param name="source">The read-only reference to reinterpret.</param>
-        <summary>Reinterprets the given read-only reference as a reference.</summary>
-        <returns>A reference to a value of type <typeparamref name="T" />.</returns>
+        <summary>Reinterprets the given read-only reference as a mutable reference.</summary>
+        <returns>A mutable reference to a value of type <typeparamref name="T" />.</returns>
         <remarks>To be added.</remarks>
```

Comment:
This deserves remarks that it is the callers responsibility to ensure the data is not mutated. Mutating a readonly reference may cause access violations or other runtime instability.

In some cases, the JIT even optimizes based on assumed immutable data and so mutating it may cause two methods to have differing views of the same data.